media/css/base/banners/pocket.scss

```diff
@@ -0,0 +1,129 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// stylelint-disable declaration-no-important
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+$brand-theme: 'firefox';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import 'includes/base';
+
+@font-face {
+    font-family: Inter;
+    font-display: swap;
+    font-weight: 700;
+    src:
+        url('https://assets.getpocket.com/web-ui/assets/Inter-Bold.c9c89216b48bee4d8727bf39d948b125.woff'),
+        url('https://assets.getpocket.com/web-ui/assets/Inter-Bold.fe38e3f307fcbac1a801c0c38dff56ef.woff2');
+}
+
+// Pocket theme variables
+$color-text-primary: #1a1a1a;
+$font-text-primary: inter, 'Helvetica Neue', helvetica, arial, sans-serif;
+
+// leave space for Close button
+$close-btn-space: calc(1rem + (2 * #{$spacing-lg}));
+
+.c-banner {
+    // banner base overrides
+    background-color: #9871ef;
+    text-align: center;
+    padding-top: 0 !important;
+    padding-bottom: 0 !important;
+
+    &-title,
+    &-content {
+        display: block;
+        width: 100%;
+    }
+
+    // protocol overrides
+    a:link {
+        color: $color-text-primary;
+    }
+
+    .mzp-c-button {
+        background-color: $color-white;
+        border-color: $color-white;
+        color: $color-text-primary;
+        font-family: $font-text-primary;
+
+        &:hover,
+        &:active {
+            background-color: $color-white;
+            border-color: $color-white;
+            color: $color-text-primary;
+            text-decoration: underline !important;
+        }
+    }
+
+    .mzp-l-content {
+        max-width: 100%;
+        padding: 0;
+    }
+
+    // end overrides
+
+    &-title {
+        @include text-title-xs;
+        box-sizing: border-box;
+        color: $color-white;
+        font-family: $font-text-primary;
+        margin-bottom: 0;
+        padding: $spacing-md $close-btn-space;
+    }
+
+    &-content {
+        padding-bottom: $spacing-md;
+    }
+}
+
+@supports (display: grid) {
+    @media #{$mq-md} {
+        // this is to match the pocket banner max width
+        $max-width-column-third: 1000px * 0.33;
+
+        .c-banner {
+            overflow-x: hidden;
+
+            &-main {
+                display: grid;
+                column-gap: $spacing-lg;
+                align-items: center;
+                grid-template-areas: '. . title cta .';
+                grid-template-columns: 1fr minmax(0, $max-width-column-third) minmax(min-content, $max-width-column-third) minmax(max-content, $max-width-column-third) minmax($close-btn-space, 1fr);
```

Comment:
this was updated from the pocket styles to always leave space for the close button